bridge/contracts/AllowTokens/AllowTokensOld.sol

```diff
@@ -5,7 +5,7 @@ pragma solidity ^0.8.0;
 import "../zeppelin/math/SafeMath.sol";
 import "../zeppelin/ownership/Ownable.sol";
 
-contract AllowTokensV0 is Ownable {
+contract AllowTokensOld is Ownable {
```

Comment:
Maybe worth to use numbers rather then aliases in names ?(in terms of consistency and versioning?)